packages/react-components/styles/colors.v2.tsx

```diff
@@ -0,0 +1,58 @@
+export enum colors {
+  greenFaint = '#97DFC1', // green disabled
+  greenBrand = '#42D689',
+  greenUI = '#1AB775',
+  goldFaint = '#E3C376', // gold disabled
+  goldBrand = '#FBCC5C',
+  goldUI = '#EEB93C',
+  goldDark = '#9C6E00',
+  beige = '#F5F4F0',
+  brownFaint = '#FFF9EE',
+  warning = '#EA6042',
+  dark = '#2E3338',
+  gray5 = '#81868B',
+  gray4 = '#9CA4A9',
+  gray3 = '#B4B9BD',
+  gray2 = '#EDEEEF',
+  gray1 = '#F8F9F9',
+  light = '#FFFFFF',
+}
+
+export default colors
+// transition table
```

Comment:
would people find a transition object useful? where it had old names but the values were linked to new colors? 
@tarikbellamine  @jeanregisser @i1skn 

otherwise I will remove

Reply:
@coreycelo